spec/system/internal/admin_awards_badges_spec.rb

```diff
@@ -23,13 +24,19 @@ def submit_form
   end
 
   it "lists the badges" do
-    Badge.all.each do |b|
-      expect(page).to have_content(b.title)
+    badges.each do |badge|
+      expect(page).to have_content(badge)
     end
   end
 
   it "awards badges" do
-    expect { submit_form }.to change { user.badges.count }.by(1).and change { user2.badges.count }.by(1)
+    expect { submit_form }.to change { user.badges.count }.by(1).
+      and change { user2.badges.count }.by(1).
+      and change(enqueued_jobs, :size).by_at_least(1)
```

Comment:
I found out that many kinds of jobs are scheduled here. What do you think about specifying a type of job (probably, related to the badges) that we want to test here?

Reply:
I added an additional case to check that notification and email jobs are queued for each badge awarded.

@lightalloy what do you think now?